content/pages/contact/contact.md

```diff
@@ -13,6 +13,9 @@ Contact our business development team at [inquiries18F@gsa.gov](mailto:inquiries
 ### Looking for a job at 18F?
 18F is part of GSA's Technology Transformation Services (TTS). View all [TTS open positions](https://join.tts.gsa.gov/) or email the talent team at [joinTTS@gsa.gov](mailto:joinTTS@gsa.gov).
 
+### Want to have 18F speak at your event?
+Check out [topics we can give talks about]({{ "/speaker-bureau/" |url }}).
```

Comment:
concern: My understanding is that, when screen readers scan links, they read the link copy.

change: I think the full sentence is clearer than the link fragment, so I'd suggest making the entire sentence the link.